user/templates/minimal-templates.md

```diff
@@ -254,6 +254,7 @@ list of packages to be installed):
   to configure it.
 - `default-mgmt-dvm`: requires `qubes-core-agent-passwordless-root` and
   `qubes-mgmt-salt-vm-connector`.
+- [Yubikey](/doc/yubikey/): You may need to install `xserver-xorg-input-all` for 2FA responses to work in web browsers like Firefox.
```

Comment:
- [Yubikey](/doc/yubikey/): You may need to install `xserver-xorg-input-libinput` for 2FA responses to work in web browsers like Firefox.